<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>462</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>462</width>
    <height>580</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial Unicode MS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>350</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Is the reading file OK?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: outset;
	border-width: 1px;
	border-radius: 15px;		
	border-color: rgb(70, 70, 70);			
	background-color: qlineargradient(spread:pad, x1:0.0113636, y1:0, x2:0, y2:1, stop:0 rgba(203, 236, 255, 255), stop:1 rgba(5, 161, 255, 255));
}
QPushButton:pressed { 			
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(150, 216, 255, 255), stop:1 rgba(5, 161, 255, 255));
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_swf_64.PNG</normaloff>J:/検査画面icon/flash_swf_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>540</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: outset;
	border-width: 1px;
	border-radius: 15px;
	border-color: rgb(70, 70, 70);
	background-color: qlineargradient(spread:pad, x1:0.0113636, y1:0, x2:0, y2:1, stop:0 rgba(254, 254, 254, 255), stop:1 rgba(165, 165, 165, 255));
}
QPushButton:pressed { 	
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(141, 141, 141, 255));
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_64.PNG</normaloff>J:/検査画面icon/flash_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>420</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>MasterFilePath</string>
   </property>
   <widget class="QLineEdit" name="lePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbHost">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>420</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>DatabaseHostName</string>
   </property>
   <widget class="QLineEdit" name="leHost">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbPort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>420</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>PortNumber(FireBird)</string>
   </property>
   <widget class="QLineEdit" name="lePort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>420</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>FileName(FDBFile)</string>
   </property>
   <widget class="QLineEdit" name="leFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbUser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>420</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>UserName</string>
   </property>
   <widget class="QLineEdit" name="leUser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbPassword">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>420</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>PassWord</string>
   </property>
   <widget class="QLineEdit" name="lePassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
